__tests__/Unit/Components/Calendar/UserSearchField.test.tsx

```diff
@@ -193,6 +193,6 @@ describe('SearchField component', () => {
         await waitFor(() => expect(input).toHaveValue(''));
 
         const noSuggestions = screen.queryAllByRole('listitem');
-        expect(noSuggestions.length).toEqual(0);
+        expect(noSuggestions).toEqual([]);
```

Comment:
How's this change making sure that progress bar won't be there if status is different from in-progress.

Reply:
This test doesn't ensure that the rendering of the progress bar will only happen when the task is "IN_PROGRESS". 
I have another primary task going on, I picked up this task because the previous owner of this task left the company.